quinn-proto/src/lib.rs

```diff
@@ -27,7 +27,6 @@ use std::{
 };
 
 mod cid_queue;
-#[doc(hidden)]
```

Comment:
Should we re-export this in quinn?

Reply:
would be nice, but not necessary from my side